.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
     "source.fixAll.eslint": true
   },
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "nuxt.isNuxtApp": false
```

Comment:
what is this for?

Reply:
i would recommend to revert it, reza had the same "problem" and also reverted it before merge.